### PR TITLE
adding "IANA Considerations" section

### PR DESCRIPTION
**Pull request by dret**
requesting "webmention" to be registered by IANA according to RFC 5988.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -395,6 +395,28 @@
...
+       
+       <dt>Relation Name:</dt>
+       <dd>preload</dd>
+       
+       <dt>Description:</dt>
+       <dd>This link relation type establishes an external resource link that
+       is used to link to a resource and declare its fetch properties.</dd>
+       
+       <dt>Reference:</dt>
+       <dd><a href="http://www.w3.org/TR/webmention/">W3C Preload
```
Comment (igrigorik) [inline index.html]: - links to wrong spec :)
- let's link to https://

File: index.html [hunk 2]
```diff
@@ -395,6 +395,28 @@
...
+  6.2.1 of [[!RFC5988]]:</p>
+     
+     <dl>
+       
+       <dt>Relation Name:</dt>
+       <dd>preload</dd>
+       
+       <dt>Description:</dt>
+       <dd>This link relation type establishes an external resource link that
+       is used to link to a resource and declare its fetch properties.</dd>
```
Comment (igrigorik) [inline index.html]: "... an external resource link for a required resource and its fetch properties"?

Comment (dret) [inline index.html]: https://github.com/dret/preload/commit/1f21ad208e0b38428e6a4d9a1f028aac98547cd5 fixes the wrong URI.

**Comment by igrigorik** [comment]
lgtm. @plehegar if you're happy, please feel free to merge.

**Comment by dret** [comment]
thanks!

**Pull request merged**